.travis.yml

```diff
@@ -15,4 +15,5 @@ script:
   - $HOME/gopath/bin/golint ./...
   - $HOME/gopath/bin/git-validation -run DCO,short-subject -v -range ${TRAVIS_COMMIT_RANGE}
   - make
+  - make test
```

Comment:
`test` contians govet, golint. There is no need to call them twice, we should remove them from travis.yml